scripts/BuildAndTest.ps1

```diff
@@ -192,10 +197,13 @@ if (-not $NoClean) {
 Install-VersionConstantsFile
 
 if (-not $NoRestore) {
-    Write-Information "Restoring NuGet packages for $SampleSolutionFile..."
-        & $NuGetExePath restore -ConfigFile $NuGetConfigFile -Verbosity $NuGetVerbosity -OutputDirectory $NuGetSamplesPackageRoot (Join-Path $SourceRoot $SampleSolutionFile)
-    if ($LASTEXITCODE -ne 0) {
-        Exit-WithFailureMessage $ScriptName "NuGet restore failed for $SampleSolutionFile."
+    if (-not $NoSamples) {
```

Comment:
I don't see an else for `-not $NoSamples` is there a reason these couldn't be combined as `if (-not $NoRestore -and -not $NoSamples)`?